portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -78,8 +78,13 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-9183 | Test Summary: Current AC plan is collapsible."
+	@ignore = "true"
 	@priority = 3
 	test CollapseCurrentPlan {
+
+		// AC Bug: Current plans card is missing
+		// AC Bug ticket: LRAC-14001
```

Comment:
Hey @joshitagaki, could you comment on the ticket saying you need to enable this test after the bug is fixed?